src/ngraph/frontend/onnx_import/op/roi_align.cpp

```diff
@@ -0,0 +1,65 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
```

Comment:
Please fix copyright date.
// Copyright 2020 Intel Corporation

Reply:
We've always used a range of dates in the license headers. @diyessi should we switch to a single year or keep the 2017-2020 format?

Reply:
It's a new file, why it should be in range from 2017?

Reply:
ngraph includes the date range of the start of ngraph to current in ALL FILES in the ngraph repo. The correct date is 2017-2020

Reply:
This has always been a convention in ngraph. To be consisten we use a range of dates throughout the project.